app/controllers/idv/session_errors_controller.rb

```diff
@@ -12,7 +12,10 @@ def warning
     private
 
     def remaining_step_attempts
-      Throttler::RemainingCount.call(user_id, :idv_resolution)
+      Throttle.for(
+        user: User.find(user_id),
```

Comment:
Worth pulling in [`EffectiveUser`](https://github.com/18F/identity-idp/blob/main/app/controllers/concerns/effective_user.rb)? Seems to encompass `user_id`, and avoids DB call if `user_id` is for the current user.

Reply:
Yes, I'd forgotten about that, thank you, fixed in de5e18cd4a288e4eda2b5eb5db10319c79daf00b

Reply:
Can we remove the `user_id` method from the class now?

Reply:
I guess so! removed in 940f5bba5fe029f0b1b0b68b38da5f59be8320c8